### ISSUE TITLE
Can't load pages or external assets that were built alongside the script

### ISSUE THREAD

**Issue by andfanilo** [whitphx/stlite#465]
I'm trying to build a MPA which loads an external image. But neither the MPA nor the external relative path work, at least on Windows 10.

The folder structure inside the folder `streamlit_app` is currently

```
├── streamlit_app.py
├── img
│   └── penguin.jpg 
└── pages
     └── another_app.py.jpg 
```

With `streamlit_app.py` being:

```python
import streamlit as st

import os
st.write(os.getcwd())

st.image("./img/penguin.jpg")
```

After dumping the app with `npm run dump streamlit_app` I can see the `pages` and `img` next to `streamlit_app.py`, but I get the following `no such file or directory` error and no pages in the sidebar, whether I serve or package it:

![image](https://user-images.githubusercontent.com/5351877/213657525-c79c2c1e-ea47-46d1-b955-7d330eeb6d43.png)

Technical details:
* Windows 10 
* Node v18.13, NPM v8.19.3
* stlite v0.22.1

I've been moving the `img` and `pages` around in the `build` folder but didn't work either and resolving the paths to absolute paths to but neither worked.
I'll try with YARN later, just in case. 
Does it work on Unix on your side? I may try with WSL2 too but I don't know if it will then work with Windows platform. Maybe by playing ith the electron-builder options to build on all platforms, it may work.

I'll keep digging tomorrow ;) but hopefully you'll find sooner than I do!

**Comment by whitphx** [comment]
Thank you for the report. I confirmed this bug occurs on Windows while it doesn't on macOS.

---
I updated the sample repository to reproduce this bug adding an image as below:
https://github.com/whitphx/stlite-desktop-example/compare/v0.3.0...v0.4.0

then tested the released executables (https://github.com/whitphx/stlite-desktop-example/releases/tag/v0.4.0)
and found the macOS version worked and Windows ver. doesn't.

macOS:
![CleanShot 2023-01-20 at 20 28 15@2x](https://user-images.githubusercontent.com/3135397/213686230-347904ee-9e2a-4279-944a-66a9e5ced290.png)

**Comment by whitphx** [comment]
My hypothesis follows:
The `walkRead()` function below which retreats all the sub directories and files in the `streamlit_app` dir does not work well for some reasons like file path separator, so the sub contents including the image file are not loaded into the Pyodide runtime.
https://github.com/whitphx/stlite/blob/9281a036634f1380482c021f4017c1f665abe7d7/packages/desktop/electron/main.ts#L60

**Comment by whitphx** [comment]
@andfanilo Could you check the console log at the verbose level and whether the image file is loaded as below?
If my hypothesis is correct, this log line does not appear on your environment.
![CleanShot 2023-01-20 at 21 08 05@2x](https://user-images.githubusercontent.com/3135397/213691890-67c6c522-1c29-4801-96bf-160ff575ee82.png)

**Comment by andfanilo** [comment]
The logs do appear but not with the same separators

![image](https://user-images.githubusercontent.com/5351877/213694664-5156a0a0-2713-48a6-8aff-df326e6803ed.png)

**Comment by andfanilo** [comment]
So yeah, can confirm that `Image.open("img\\penguin.jpg")` does work, so I guess all folders are flattened

![image](https://user-images.githubusercontent.com/5351877/213699126-53a455a4-41a6-4d2c-9b14-aa22bb5e3b74.png)

**Comment by whitphx** [comment]
Oh, thank you!
I should convert the path separator when the host OS is Windows.

**Comment by whitphx** [comment]
This should be fixed with the next release, v0.22.2
